### PR TITLE
Ubuntu exceptions 

### PR DESCRIPTION
**Pull request by dalesit**
Ubuntu uses /etc/apache2/conf-enabled for custom config. This adds an exception for Ubuntu in the Debian family case

### CODE REVIEW

**Comment by jyaworski** [comment]
Hello. Please rebase and squash.

**Comment by dalesit** [comment]
Rebased and squashed

File: manifests/params.pp [hunk 1]
```diff
@@ -8,9 +8,14 @@
 
   case $::osfamily {
     'Debian': {
+      if $::operatingsystem == ubuntu {
```
Comment (jyaworski) [inline manifests/params.pp]: Should this be == "Ubuntu"?

Comment (igalic) [inline manifests/params.pp]: yesno… string comparison in puppet are caseinsensitive.

Comment (dalesit) [inline manifests/params.pp]: If you are using a bare word, then it needs to begin with lowercase.

see [https://docs.puppetlabs.com/puppet/4.2/reference/lang_data_string.html#bare-words](https://docs.puppetlabs.com/puppet/4.2/reference/lang_data_string.html#bare-words)

so, can be either format, but `== Ubuntu` would be incorrect

**Comment by jyaworski** [comment]
Since we don't have spec tests....

Is this also compatible with 3.x? I would assume so, but the bare words doc doesn't exist on 3.8 (by the same name).

**Comment by dalesit** [comment]
It is compatible with 3.x with future parser
[https://docs.puppetlabs.com/puppet/3.8/reference/future_lang_data_string.html#bare-words](https://docs.puppetlabs.com/puppet/3.8/reference/future_lang_data_string.html#bare-words)

**Pull request merged**